kustomize/kustomize.go

```diff
@@ -281,9 +282,13 @@ func (km *Manager) Kustomize(namespace string, objects ...runtime.Object) ([]run
 }
 
 func GetUnstructuredObjects(data []byte) ([]runtime.Object, error) {
+	utfData, convertToUTFErr := BytesToUtf8Lf(data)
```

Comment:
Just a question... why do we need to have a specific err name?

Reply:
Handling on fallthroughs generally makes me nervous. At least this is compiler enforced.

Reply:
The go way is not to continue after error you immediatel  return, in which reusing the err is the convention

Reply:
I'll change it (and will uphold it going forward), but wouldn't mind an explanation/story of why the community settled on this. Every time I write go I feel like it's a bug waiting to happen.